README.md

```diff
@@ -20,13 +15,13 @@ on:
       - master
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: self-hosted
     steps:
     - uses: actions/checkout@v2
       with:
         fetch-depth: '0'
     - name: Bump version and push tag
-      uses: anothrNick/github-tag-action@1.36.0
+      uses: ext-fork-github-tag-action@1.0.0
```

Comment:
FYI: This tag will get updated in a secondary cleanup PR once I have an actual tag and a PR against the SDS RW repo.